index.html

```diff
@@ -200,98 +200,98 @@
-    <dfn data-cite="!INFRA#list-iterate">for each</dfn>,
-    <dfn data-cite="!INFRA#iteration-continue">continue</dfn>,
-    <dfn data-cite="!INFRA#list-is-empty">is empty</dfn>,
-    <dfn data-cite="!INFRA#list-is-not-empty">is not empty</dfn>,
-    <dfn data-cite="!INFRA#list-append" data-lt="list-append">append</dfn>,
-    <dfn data-cite="!INFRA#list-contain" data-lt="list-contain">contains</dfn>,
-    <dfn data-cite="!INFRA#parse-json-from-bytes">parse JSON from bytes</dfn> and
-    <dfn data-cite="!INFRA#serialize-json-to-bytes">serialize JSON to bytes</dfn>,
-    are defined in [[!INFRA]].
-  </p>
-  <p>
-    <dfn data-cite="!DOM#concept-event-fire">fire an event</dfn>,
-    <dfn data-cite="!DOM#interface-event">Event</dfn>,
-    <dfn data-cite="!DOM#dictdef-eventinit">EventInit</dfn>,
-    <dfn data-cite="!DOM#eventtarget">EventTarget</dfn>,
-    <dfn data-cite="!DOM#abortsignal">AbortSignal</dfn>,
-    <dfn data-cite="!DOM#abortsignal-aborted-flag">aborted flag</dfn>, and
-    <dfn data-cite="!DOM#abortsignal-add">add the following abort steps</dfn>
-    are defined in [[!DOM]].
+    [[RFC5234]].
+  </p>
+  <p>
+    <dfn data-cite="WEBIDL#dfn-throw"><code>throw</code></dfn>,
+    <dfn data-cite="WEBIDL#dfn-create-exception"><code>creating</code></dfn>,
+    <dfn data-cite="WEBIDL#idl-DOMString"><code>DOMString</code></dfn>,
+    <dfn data-cite="WEBIDL#idl-dictionary"><code>Dictionary</code></dfn>,
+    <dfn data-cite="WEBIDL#idl-ArrayBuffer"><code>ArrayBuffer</code></dfn>,
+    <dfn data-cite="WEBIDL#common-BufferSource"><code>BufferSource</code></dfn>,
+    <dfn data-cite="WEBIDL#idl-any"><code>any</code></dfn>,
+    <dfn data-cite="WEBIDL#dfn-present">not present</dfn>,
+    <dfn data-cite="WEBIDL#exceptiondef-typeerror"><code>TypeError</code></dfn>,
+    <dfn data-cite="WEBIDL#idl-DOMException"><code>DOMException</code></dfn>,
+    <dfn data-cite="WEBIDL#aborterror"><code>AbortError</code></dfn>,
+    <dfn data-cite="WEBIDL#syntaxerror"><code>SyntaxError</code></dfn>,
+    <dfn data-cite="WEBIDL#notsupportederror"><code>NotSupportedError</code></dfn>,
+    <dfn data-cite="WEBIDL#networkerror"><code>NetworkError</code></dfn>,
+    <dfn data-cite="WEBIDL#notreadableerror"><code>NotReadableError</code></dfn>,
+    <dfn data-cite="WEBIDL#timeouterror"><code>TimeoutError</code></dfn>, and
+    <dfn data-cite="WEBIDL#notallowederror"><code>NotAllowedError</code></dfn>,
+    are defined in [[WEBIDL]].
+  </p>
+  <p>
+    <dfn data-cite="ECMASCRIPT#sec-promise-objects"><code>Promise</code></dfn>,
+    <dfn data-cite="ECMASCRIPT#sec-json-object">JSON</dfn>,
+    <dfn data-cite="ECMASCRIPT#sec-json.stringify">JSON.stringify</dfn>,
+    <dfn data-cite="ECMASCRIPT#sec-json.parse">JSON.parse</dfn> and
+    <dfn data-cite="ECMASCRIPT#sec-object-internal-methods-and-internal-slots">internal slots</dfn>
+    are defined in [[ECMASCRIPT]].
+  </p>
+  <p>
+    <dfn data-cite="INFRA#ascii-decode">ASCII decode</dfn>,
```

Comment:
Delete all these

Reply:
@marcoscaceres, do I need to add the "INFRA" to the `<body>`? It's odd, as without it in the `<body>`, all the `INFRA` relevant links disappeared. e.g. `[=list/For each=]` throws ReSpec error.

While this does not happen in WakeLock, did I miss something critical?

Reply:
I found the root cause, `xref: "web-platform"` is required in `respecConfig`. :)

Reply:
was just about to reply :) Exactly.

Reply:
so, here is what specs "web-platform" maps to: 
https://github.com/w3c/respec/blob/84fef3fea275143349a313a888546e8f2cb84137/src/core/xref.js#L24

I'm thinking we should add Encoding and MIME there.

Reply:
Thanks, good to know that!

Reply:
> `typedef (DOMString or unrestricted double or ArrayBuffer or Dictionary) NDEFRecordData;
`

@marcoscaceres, the `Dictionary` type in above IDL could not be recognized, any idea?

Reply:
Dictionary is not a type :) You need an actually type.

Reply:
Put differently, what "dictionary"? Or do you mean "object"... 

See https://heycam.github.io/webidl/#prod-DistinguishableType

Reply:
It expects a JSON object, and does not indicate any "dictionary". For this situation, can it be an "Object"? I'm not quite sure, tbh.

Reply:
no, it would have to be "object".